core/services/eth/pool_test.go

```diff
@@ -2,35 +2,39 @@ package eth_test
 
 import (
 	"context"
-	"errors"
 	"math/big"
 	"net/http/httptest"
 	"net/url"
 	"testing"
+	"time"
 
 	"github.com/ethereum/go-ethereum/common/hexutil"
 	"github.com/ethereum/go-ethereum/rpc"
-	"github.com/stretchr/testify/assert"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/mock"
 	"github.com/stretchr/testify/require"
 	"github.com/tidwall/gjson"
-	"go.uber.org/multierr"
 
 	"github.com/smartcontractkit/chainlink/core/internal/cltest"
 	"github.com/smartcontractkit/chainlink/core/logger"
-	. "github.com/smartcontractkit/chainlink/core/services/eth"
+	"github.com/smartcontractkit/chainlink/core/services/eth"
+	ethmocks "github.com/smartcontractkit/chainlink/core/services/eth/mocks"
 )
 
+func init() {
+	eth.DialRetryInterval = 100 * time.Millisecond
+}
```

Comment:
Could this be done from a `package eth` test file, like `helpers_test.go`, so that the var doesn't need to be exported?